packages/preview-middleware-client/test/unit/cpe/changes/service.test.ts

```diff
@@ -336,10 +312,10 @@ describe('SelectionService', () => {
             })
         );
 
-        expect(window.fetch).toHaveBeenLastCalledWith('/preview/api/changes', {
-            body: '{"fileName":"id_1640106755570_203_propertyChange"}',
-            headers: { 'Content-Type': 'application/json' },
-            method: 'DELETE'
+        expect(fetchMock).toHaveBeenLastCalledWith('/preview/api/changes', {
+            'body': '{"fileName":"id_1640106755570_203_propertyChange"}',
```

Comment:
Not a real problem but why did you put the property names in quotes?

Reply:
When i was trying out I thought matcher was not matching and I added quotes then forgot remove it.